src/Parser/DemonHunter/Vengeance/Modules/Spells/DemonSpikes.js

```diff
@@ -40,6 +41,18 @@ class DemonSpikes extends Analyzer {
     return formatPercentage(this.hitsWithDS / (this.hitsWithDS + this.hitsWithoutDS));
   }
 
+  suggestions(when) {
+    const hitsWithDSOffCDPercent = this.hitsWithDSOffCD / (this.hitsWithDS+ this.hitsWithoutDS);
+    when(hitsWithDSOffCDPercent).isGreaterThan(0.1)
+      .addSuggestion((suggest, actual, recommended) => {
+        return suggest(<span> Cast <SpellLink id={SPELLS.DEMON_SPIKES.id} /> more regularly while actively tanking the boss or when they use a big phsyical attack. You missed having it up for ${formatPercentage(hitsWithDSOffCDPercent)}% of physical hits.</span>)
```

Comment:
extra $ here, looks like a holdover from template strings. after this it looks good

Reply:
Also that should technically be `React.Fragment` instead of `span`.